packages/nextjs/app/builders/0x729a2043F6702Ad76A62215A69BC00360E290268/page.tsx

```diff
@@ -0,0 +1,268 @@
+
+const SOCIAL_LINKS = [
+  { href: "https://github.com/anynomousfriend", label: "GitHub", icon: "💻" },
+  { href: "https://x.com/SsubhankarX", label: "X", icon: "🐦" },
+  { href: "https://www.linkedin.com/in/subh-choudhury/", label: "LinkedIn", icon: "💼" },
+  { href: "https://t.me/Sshubhankar", label: "Telegram", icon: "✈️" },
+  { href: "https://subhankar-s-portfolio.vercel.app/", label: "Portfolio", icon: "🌐" },
+];
+
+const SKILLS = ["Solidity", "TypeScript", "Hardhat", "Next.js", "Figma", "UI Design", "Python", "JavaScript", "Linux"];
+
+// Custom Address Display Component (no network calls)
+const AddressDisplay = ({ address }: { address: string }) => {
+  const [copied, setCopied] = useState(false);
+
+  const handleCopy = async () => {
+    try {
+      await navigator.clipboard.writeText(address);
+      setCopied(true);
+      setTimeout(() => setCopied(false), 2000);
+    } catch (err) {
+      console.error("Failed to copy:", err);
+    }
+  };
+
+  const shortAddress = `${address.slice(0, 6)}...${address.slice(-4)}`;
+
+  return (
+    <div className="flex items-center gap-2 justify-center">
+      <button
+        onClick={handleCopy}
+        className="font-mono text-xs bg-base-200 px-3 py-2 rounded-lg hover:bg-base-300 transition-colors border border-base-300 flex items-center gap-2"
+        title={address}
+      >
+        <span className="text-primary">●</span>
+        <span>{shortAddress}</span>
+        <span className="text-base-content/50">{copied ? "✓" : "📋"}</span>
+      </button>
+    </div>
+  );
+};
+
+// Enhanced Matrix-style floating particles
+const MatrixParticles = () => {
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  if (!mounted) return null;
+
+  const particles = Array.from({ length: 50 }, (_, i) => ({
+    id: i,
+    left: `${(i * 2) % 100}%`,
+    delay: `${(i * 0.3) % 5}s`,
+    duration: `${6 + (i % 3)}s`,
+    char: ["0", "1", "{", "}", "[", "]", "→", "λ", "Ξ"][i % 9],
+    size: i % 3 === 0 ? "text-sm" : i % 2 === 0 ? "text-base" : "text-xs",
+  }));
+
+  return (
+    <div className="absolute inset-0 overflow-hidden pointer-events-none">
+      {particles.map(particle => (
+        <div
+          key={particle.id}
+          className={`absolute ${particle.size} font-mono text-primary animate-matrix-fall`}
+          style={
+            {
+              left: particle.left,
+              top: "-20px",
+              animationDelay: particle.delay,
+              animationDuration: particle.duration,
+              "--matrix-distance": "110vh",
+              opacity: 0.7,
+            } as React.CSSProperties
+          }
+        >
+          {particle.char}
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const SubhankarPage: NextPage = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    setIsVisible(true);
+  }, []);
```

Comment:
Since we're using Next.js, you'll notice the page loads faster without this behavior. Try it yourself and compare the experiences.